packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.component.tsx

```diff
@@ -26,6 +26,8 @@ export const SiderailNavButton: React.FC<SiderailNavButtonProps> = ({
 }) => {
   const layout = useLayoutType();
   const { workspaces, workspaceWindowState } = useWorkspaces();
+  console.log('workspaces: ', workspaces);
+  console.log('workspaceWindowState: ', workspaceWindowState);
```

Comment:
```suggestion
```

Reply:
Yikes, eslint should have caught that.

Reply:
I know, right? Do you have an idea why it didn't?

Reply:
Yep, looks like that rule wasn't enabled. Working on a PR to fix that now.